packages/create-payload-app/src/lib/manage-env-files.ts

```diff
@@ -5,44 +5,68 @@ import type { CliArgs, DbType, ProjectTemplate } from '../types.js'
 
 import { debug, error } from '../utils/log.js'
 
-const updateEnvVariables = (
-  contents: string,
-  databaseType: DbType | undefined,
-  databaseUri: string,
-  payloadSecret: string,
-): string => {
+const updateEnvExampleVariables = (contents: string, databaseType: DbType | undefined): string => {
   return contents
     .split('\n')
-    .filter((e) => e)
     .map((line) => {
       if (line.startsWith('#') || !line.includes('=')) {
-        return line
+        return line // Preserve comments and unrelated lines
       }
 
-      const [key, ...valueParts] = line.split('=')
-      let value = valueParts.join('=')
-
-      if (
-        key === 'MONGODB_URI' ||
-        key === 'MONGO_URL' ||
-        key === 'DATABASE_URI' ||
-        key === 'POSTGRES_URL'
-      ) {
-        value = databaseUri
-        if (databaseType === 'vercel-postgres') {
-          value = databaseUri
+      const [key] = line.split('=')
+
+      // Handle placeholder connection strings based on databaseType
+      if (key === 'DATABASE_URI' || key === 'POSTGRES_URL' || key === 'MONGODB_URI') {
+        switch (databaseType) {
+          case 'mongodb':
+            return `DATABASE_URI=mongodb://your-connection-string-here`
```

Comment:
All of these should be more realistic. Use `dbChoiceRecord`'s `dbConnectionPrefix` from `lib/select-db.ts`.

```
mongodb://127.0.0.1/payload-project
postgres://postgres:<password>@127.0.0.1:5432/payload-project
```

Reply:
Done